tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx

```diff
@@ -0,0 +1,92 @@
+import {
+  screen,
+  render,
+  fireEvent,
+  waitForElementToBeRemoved
+} from '@testing-library/react'
+import SecurityBlock from '~/containers/tutor-profile/security-block/SecurityBlock'
+
+const changeInputValue = (label, value) => {
+  fireEvent.change(screen.getByLabelText(label), { target: { value } })
+}
+
+describe('SecurityBlock', () => {
+  beforeEach(() => {
+    render(<SecurityBlock />)
+  })
+
+  it('renders title and description', () => {
+    expect(
+      screen.getByText('editTutor.passwordSecurityTab.title')
+    ).toBeInTheDocument()
+    expect(
+      screen.getByText('editTutor.passwordSecurityTab.description')
+    ).toBeInTheDocument()
+  })
+
+  it('resets form when discard button is clicked', () => {
+    changeInputValue(
+      'editTutor.passwordSecurityTab.currentPassword',
+      'oldPassword'
+    )
+    changeInputValue('editTutor.passwordSecurityTab.newPassword', 'newPassword')
+    changeInputValue(
+      'editTutor.passwordSecurityTab.retypePassword',
+      'newPassword'
+    )
+
+    fireEvent.click(screen.getByText('common.discard'))
+
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.currentPassword')
+    ).toHaveValue('')
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.newPassword')
+    ).toHaveValue('')
+    expect(
+      screen.getByLabelText('editTutor.passwordSecurityTab.retypePassword')
+    ).toHaveValue('')
```

Comment:
do you need to check for 3 labels? I mean, maybe 1 is enough

Reply:
You're right, I changed it